cmd/crank/render/xrd_test.go

```diff
@@ -0,0 +1,82 @@
+package render
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+)
+
+func TestDefaultValues(t *testing.T) {
+	type args struct {
+		xr  map[string]any
+		crd extv1.CustomResourceDefinition
+	}
+
+	cases := map[string]struct {
+		name    string
+		args    args
+		want    map[string]any
+		wantErr bool
+	}{
+		"SetDefaultValues": {
```

Comment:
another good test case here would be that default values don't overwrite values already set in the XR